smart_open/webhdfs.py

```diff
@@ -158,15 +181,17 @@ def detach(self):
 
     def _upload(self, data):
         payload = {"op": "APPEND"}
-        init_response = requests.post("http://" + self.uri_path,
+        init_response = requests.post(self._uri,
```

Comment:
Please use a hanging indent here. Or, even better, put everything on one line, it should fit.

Reply:
88 chars in line - it is the sign that smart_open should start using [black](https://github.com/psf/black) :)

Reply:
Over my dead body ;)